src/main/java/kafdrop/service/KafkaMonitorImpl.java

```diff
@@ -280,10 +280,10 @@ public SearchResultsVO searchMessages(String topic,
         case EXCEEDED_MAX_SCAN_COUNT:
           results.setCompletionDetails(
             String.format(
-              "Search timed out after scanning %d messages.  Last scanned message timestamp was %d.  Adjust your time" +
-                " span for more results.",
+              "Search timed out after scanning %d messages. Last scanned message timestamp was %s. Adjust your time" +
+              " span for more results.",
```

Comment:
Can't we just fix the format?
```suggestion
              "Search timed out after scanning %1$d messages. Last scanned message timestamp was %2$tF %2$tT. Adjust your time" +
                " span for more results.",
```